scripts/js-native-class.rb

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+require 'open3'
+
+CONTROL_CHARS = /\e\[[^\x40-\x7E]*[\x40-\x7E]/
+
+def stream_and_capture(*cmd)
+  status = nil
+  stdout_str = ''
+  stderr_str = ''
+
+  Open3.popen3(*cmd) do |stdin, stdout, stderr, wait_thr|
+    Thread.new do
+      while char = stdout.getc
+        $stdout.write char
+        stdout_str << char
+        $stdout.flush
+      end
+    end
+  
+    # For stderr
+    Thread.new do
+      while char = stderr.getc
+        $stderr.write char
+        stderr_str << char
+        $stderr.flush
+      end
+    end
+
+    status = wait_thr.value
+  end
+  [status, stdout_str, stderr_str]
+end
+
+Dir.chdir("repo")
+
+files = if File.exist?("plugin.rb")
+  Dir["{test,assets,admin/assets}/javascripts/**/*.js"]
+else
+  Dir["{test,javascripts}/**/*.js"]
+end
+
+if files.empty? && files.none? { |f| File.read(f).includes(".extend") }
```

Comment:
Right?

```suggestion
if files.empty? || files.none? { |f| File.read(f).includes(".extend") }
```

Reply:
Oh yes... that'll be why it's running for every repo right now 😂 

thanks!